playbooks/tests/tasks/integration.yml

```diff
@@ -105,6 +105,43 @@
     volume: turtle-vol
   when: cinder.enabled|bool
 
+- name: instance is active
```

Comment:
what happens if instance take little longer to go to active state?
probably we should have retry logic, 6 times with 10sec delay between each retry.